package.json

```diff
@@ -246,7 +250,7 @@
   },
   "husky": {
     "hooks": {
-      "pre-push": "yarn checkcommit && yarn documentation && pretty-quick --check"
+      "pre-push": "yarn check-commit && yarn documentation"
```

Comment:
this is definitely a temporary measure, until I receive some guidance on committing/developing on Windows OS